wordpress-mobile/WordPress-iOS
# Updates CocoaLumberjack to current & fixes logging in WordPressKit

Related: #7361 

This removes all of the Swift custom wrappers and uses CocoaLumberjack directly now that it has Swift support. I also simplified the Podfile putting the WordPress target as the root target and inheriting only search paths in the two extensions. This appears to have improved build times as well.

I'd still like to get rid of CocoaLumberjack but it's working well enough now with the Swift support.

To test:
1. Clean, build, run.
2. Make sure logging works properly.
3. Find something in WordPressKit that you can fire off (like in AccountService) and add some custom logging in there.
4. Cause the action in step 3 to happen. Verify custom log message makes it way to the console and the app log in Me > Help & Support > Activity Logs.

Needs review: @bummytime, @elibud
